integration/test/sameclusterid/project.yaml

```diff
@@ -0,0 +1,24 @@
+version: 1
+
+test:
+  env:
+    - "VAULT_TOKEN=mytoken"
```

Comment:
When this could be ordered alphabetically, this would be really nice.

Reply:
Done.